packages/next-app/src/app/internalmanager/[network]/[operationKind]/[tokenAddress]/page.tsx

```diff
@@ -85,6 +88,22 @@ export default function Page({
     token: params.tokenAddress,
   });
 
+  useEffect(() => {
+    const tokenInfo = internalBalanceTokenData?.user
+      ?.userInternalBalances?.[0] || {
+      balance: "0",
+      tokenInfo: {
+        __typename: "Token",
+        address: params.tokenAddress,
+        decimals: walletAmount?.decimals || 0,
+        symbol: walletAmount?.symbol || "",
+        name: walletAmount?.symbol || "",
+      },
```

Comment:
It sounds like this would have the same effect of assigning to the variable outside of the useEffect if you're not setting tokenData elsewhere